### DISCUSSION TITLE
xrdp bugging with browsers

### DISCUSSION THREAD

**Discussion by Necrosis000** [neutrinolabs/xrdp#3272] [General]
so when I connect to my raspberry pi and use an internet browser like chromium and firefox look like this. everything else is fine but this happens and i dont know why

![screenshot](https://github.com/user-attachments/assets/0208ce41-9d22-48eb-86aa-2ef1378a31b7)

**Comment by matt335672** [comment]
See #2869 / #2851 / #2430
